### PR TITLE
Adding percona manifest and readme 

### PR DESCRIPTION
**Pull request by Adarshkumar14**

### CODE REVIEW

File: k8s-manifests/percona-mongo/README.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+# HCE-Percona MongoDb Operator Integration
+Based on the best practices for deployment and configuration, Percona Distribution for MongoDB Operator contains everything we need to quickly and consistently deploy and scale Percona Server for MongoDB instances into a Kubernetes cluster on-premises or in the cloud.The Operator contains the necessary Kubernetes settings to maintain a consistent Percona Server for MongoDB instance and for modification or deletion of items in your Percona Server for MongoDB environment. 
+
+- You can create a Percona Server for MongoDB environment with no single point of failure and the ability to span multiple activity or availability zones.
+
+- You can scale horizontally with sharding or by altering the size of the nodes in the replica set through Custom Resource options.
+
+- You can configure MongoDB cluster backups to run on a scheduled basis. Backups are stored on an external S3-compatible object storage.
+
+- You can Rely on support for data encryption in transit.
```
Comment (Jonsy13) [inline k8s-manifests/percona-mongo/README.md:10]:
```suggestion
- You can rely on support for data encryption in transit.
```

File: k8s-manifests/percona-mongo/README.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+Kubectl apply -f https://hce.chaosnative.com/manifests/latest/percona-mongo/percona-mongodb-crd.yaml
+```
+
+Output:
+```bash
+customresourcedefinition.apiextensions.k8s.io/perconaservermongodbs.psmdb.percona.com created
+customresourcedefinition.apiextensions.k8s.io/perconaservermongodbbackups.psmdb.percona.com created
+customresourcedefinition.apiextensions.k8s.io/perconaservermongodbrestores.psmdb.percona.com created
+```
+-  Setting up the RBAC  for Percona server for MongoDB\
```
Comment (Jonsy13) [inline k8s-manifests/percona-mongo/README.md:38]:
```suggestion
-  Setting up the RBAC for Percona server for MongoDB
```

File: k8s-manifests/percona-mongo/percona-secret.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-cluster-name-backup-s3
+  namespace: litmus
+type: Opaque
+stringData:
+  AWS_ACCESS_KEY_ID: XXXXXXXXXXXXXXXXXXXXXXXX
+  AWS_SECRET_ACCESS_KEY: XXXXXXXXXXXXXXXXXXXX
+---
```
Comment (Jonsy13) [inline k8s-manifests/percona-mongo/percona-secret.yaml:29]: Can we add it in different file

File: k8s-manifests/percona-mongo/README.md (hunk 2 above)
Comment (Adarshkumar14) [reply k8s-manifests/percona-mongo/README.md:38]: I have added `\` this for new line

**Review by Jonsy13 — APPROVED** [review]
Looks good!

**Pull request merged**